src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -248,6 +248,11 @@ public class KinesisClientLibConfiguration {
     private String applicationName;
     private String tableName;
     private String streamName;
+
+    /**
+     * @return stream ARN
```

Comment:
FYI: `@return` on a member variable is not an expected annotation, and IDEs may flag it as "not allowed here". Correct variable Javadoc is something like:
```
/**
 * <description>
 */
```

Reply:
[Lombok @Getter](https://projectlombok.org/features/GetterSetter) copies @return javadoc from annotated member vars to generated getters. But since its not consistent with the other variables in this class I'll remove it.